docs/api.md

```diff
@@ -1306,6 +1306,7 @@ await page.tracing.stop();
 - `options` <[Object]>
   - `path` <[string]> A path to write the trace file to. **required**
   - `screenshots` <[boolean]> captures screenshots in the trace.
+  - `categories` <[Array]<[string]>> specify custom categories to use instead of default. 
```

Comment:
is there any good resource to link to about tracing categories?

Reply:
There isn't. Various tools just have a list hardcoded most of the time. You normally find what categories you want by reading devtools/lighthouse/webpagetest source. :)